fastlane/Pluginfile

```diff
@@ -6,7 +6,7 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 3.2.0'
 end
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.14.0'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', ref: 'aefec479766afe93ef30696c1c0ca82e91574ecf'
```

Comment:
As you mentioned in your PR description, this needs to be updated once we do a new release. Just highlighting this as a comment so we don't forget before merge.